uni/lib/view/common_widgets/pages_layouts/general/widgets/refresh_state.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:uni/model/providers/startup/profile_provider.dart';
+import 'package:uni/model/providers/startup/session_provider.dart';
+
+class RefreshState extends StatelessWidget {
```

Comment:
This class is not reusable and does not have state, so should not be a widget, IMO. Simply inline it

Reply:
I don't know if that's the best approach, because it's used in two separate places (`GeneralPageViewState` and `SecondaryPageViewState`) and its contents aren't the simplest, so inlining it would make the code quite messy.

Reply:
It's ok to stay as a method, then, I hope.

Reply:
@Process-ing have you looked into this?

Reply:
Supposedly, we decided in a meeting that this was ok, did you want me to change something still?